### PR TITLE
Add an article with the answers to HN questions

### PR DESCRIPTION
**Pull request by almet**
r? @n1k0 @Natim

### CODE REVIEW

File: content/2015.12.hacker-news.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+milestone for the project, even if it also comes with some sort of pressure and
+stress :)
+
+Since a few questions emerged from the thread there, we thought it'd
+appropriate to provide some answers here.
+
+What does this name mean?
+=========================
+
+Kinto-un is the name of the flying nimbus in Dragon ball. It's said that
```
Comment (Natim) [inline content/2015.12.hacker-news.rst]: `Flying Nimbus`

File: content/2015.12.hacker-news.rst [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+"the cloud can only be ridden by someone with a pure heart". Oh, well.
+    
+Maintaining a service is hard.
```
Comment (Natim) [inline content/2015.12.hacker-news.rst]: I wouldn't put a dot at the end of titles.

File: content/2015.12.hacker-news.rst [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+==============================
+
+The main benefit of Firebase and Parse is that you don't have to run them.
+
+That is true. We want to make it very easy to deploy / use. We can even
+imagine to allow developers to run a Kinto instance in one-click on
+DigitalOcean, Heroku or Bitnami.
```
Comment (Natim) [inline content/2015.12.hacker-news.rst:33]: Should we add a link to what we've started on Alwaysdata: https://github.com/Natim/auto-kinto-alwaysdata ?

File: content/2015.12.hacker-news.rst [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+When a record is modified, Kinto sends a notification on a pubsub channel. We
+integrated Pusher.com and WebPush very recently.  Currently, the notification
+payload contains the old and new version of the record. We plan on making
+improvements, like sending diffs only. #babysteps #ftw! 
+      
+Also, the current mecanism does not allow to send write operations via
+websockets. Those are sent via the REST API.  
+
+You're missing an administration panel !
```
Comment (Natim) [inline content/2015.12.hacker-news.rst]: In english we shall remote space before `!`

File: content/2015.12.hacker-news.rst [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+========================================
+
+That is true, but `we're working on it pretty hard
+<https://github.com/Kinto/kinto-admin >`. If you want, you can have a look at
```
Comment (Natim) [inline content/2015.12.hacker-news.rst]: _ missing to build a link?

File: content/2015.12.hacker-news.rst [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+it now, even though it's fairly new.
```
Comment (Natim) [inline content/2015.12.hacker-news.rst]: Ready this make me feel kinto is not ready for me to use yet. Everything looks new and not ready (admin notifications, etc)

File: content/2015.12.hacker-news.rst [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+
+Since the two solutions are solving slightly different problems, we will make
+our best to guide the users towards the best for their use-case.  `We also plan
+to extend this comparison table <https://github.com/Kinto/kinto/issues/330>`_
+to compare products rather than server solutions.
+
+We are also developing X / I wonder how it compares with X
+==========================================================
+
+A few years ago, we built Daybed, a similar project which never had the chance
```
Comment (Natim) [inline content/2015.12.hacker-news.rst]: Link to the Daybed project?

File: content/2015.12.hacker-news.rst [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+      
+We realize there are many many alternatives in the wild, like http://gun.js.org
+or http://kuzzle.io just to mention FOSS. Of course, Parse could also be
+released under an open-source license some day.
+
+For us, it mainly means that Kinto was a good idea!
+      
+We are a team of 5, and we don't pretend to knock down the tech of big
+companies. And our top management is not even aware of this project. We build
+Kinto with some ideals : a simple solution for simple needs that is fully
```
Comment (Natim) [inline content/2015.12.hacker-news.rst]: in english no spaces before semi-columns.

File: content/2015.12.hacker-news.rst (hunk 6 above)
Comment (almet) [inline content/2015.12.hacker-news.rst]: I've rephrased this, then.

File: content/2015.12.hacker-news.rst [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+
+Or if someone runs a public Kinto instance and wants to charge for its usage,
+that's completely fine!
+
+At Mozilla, we just can't rely on third parties like Firebase or Parse for
+critical data.
+
+If you read french, we've written `an article about our vision on self-hosting
+<http://www.servicedenuages.fr/difficile-promesse-autohebergement>`_
+(yes, we need to translate it!)
```
Comment (n1k0) [inline content/2015.12.hacker-news.rst]: Nit: suggest we actually need _help_ translating it? ;)

File: content/2015.12.hacker-news.rst [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+Realtime updates, what about Kinto?
+===================================
+
+When a record is modified, Kinto sends a notification on a pubsub channel. We
+integrated Pusher.com and WebPush very recently.  Currently, the notification
+payload contains the old and new version of the record. We plan on making
+improvements, like sending diffs only. #babysteps #ftw! 
+      
+Also, the current mecanism does not allow to send write operations via
+websockets. Those are sent via the REST API.
```
Comment (n1k0) [inline content/2015.12.hacker-news.rst]: We should clarify if we ever plan on doing so.

Comment (almet) [inline content/2015.12.hacker-news.rst]: Those are only sent via the REST API. We were recently talking
about abstracting the protocol into something different, but it's
undecided as of now.

That clarifies it?

Comment (n1k0) [inline content/2015.12.hacker-news.rst]: Yup, so people know it's considered.

**Comment by n1k0** [comment]
Looks good. I've seen a question about offline-first support, I think it's worth adding a specific section mentioning Kinto.js.

**Comment by Natim** [comment]
People were also asking question about usage in production, maybe we should clarify that we are using it in production?

**Comment by almet** [comment]
@natim I've added two sections about offline first and production. Can you please review them and tell me if that's okay? Thanks!

File: content/2015.12.hacker-news.rst [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+- The (now dead) reading-list project was using one of the first versions of
+  Kinto
+- We have made `Syncto <https://github.com/mozilla-services/syncto>`_:
+  a service which exposes the API of Kinto and talks with Firefox Sync in the
+  background, so that FirefoxOS developers can use the Kinto.js library
+- We are currently doing the distribution of fonts for Fennec Over The Air
+  using Kinto.
+
+Also, it's not in production yet, but we will use Kinto for the Certificate
+Revocation Lists in the future.
```
Comment (Natim) [inline content/2015.12.hacker-news.rst:61]: as well as for the Addons, Plugins and Graphical driver Blocklist in Firefox.

**Comment by n1k0** [comment]
LGTM

File: content/2015.12.hacker-news.rst [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+
+Offline-first support
+=====================
+
+One of the core ideas behind `Kinto.js <https://github.com/Kinto/kinto.js>`_ is
+that it should be offline first: all operations can be done when online or
+offline, and the data is synchronized when the connection goes back to normal.
```
Comment (Natim) [inline content/2015.12.hacker-news.rst]: the data are

Comment (Natim) [inline content/2015.12.hacker-news.rst]: Should we add a note about Android and IOS offline first clients? refs https://github.com/Kinto/kinto/issues/327

**Comment by almet** [comment]
Okay, I'm about to publish this. Thanks all for the quick feedback!

**Pull request merged**